src/app/related-search/related-search.component.html

```diff
@@ -1,7 +1,17 @@
 <div *ngIf="results?.length > 0" class="card">
     <h3>Searches related to {{this.keyword}}</h3>
     <div *ngFor="let result of results" class="related">
-      <a [routerLink]="resultsearch" [queryParams]="{query: result.label}" class="rellink">{{result.label}}</a>
+      <a [routerLink]="resultsearch" [queryParams]="{query: result.label,
+                                                      verify: false,
+                                                      nav: 'filetype,protocol,hosts,authors,collections,namespace,topics,date',
+                                                      start: 0,
+                                                      indexof: 'off',
+                                                      meanCount: '5',
+                                                      resource: 'global',
+                                                      prefermaskfilter: '',
+                                                      rows: 10,
```

Comment:
@harshit98 No, I can't, result.label cannot be pre-defined in a .ts file, result is only defined in the html file, so the only way is to call a function on (click) and then set the value of query, and then pass the queryParams. @nikhilrayaprolu What do you think?